sass/pages/_home.scss

```diff
@@ -1 +1,111 @@
-//This styles are specific styles for home.php
\ No newline at end of file
+//This styles are specific styles for home.php
+.latest_post{
+
+  @include pink_background_gradient;
+
+    .latest_post_wrapper{
+      padding: 3rem 1rem 2rem 1rem;
+
+      .latest_post_image{
+        @include image_box_shadow;
+      }
+    }
+  }
+
+  .post_content{
+    padding-top: 3rem;
+    padding-bottom: 1rem;
+  }
+
+  .all_posts{
+    background: linear-gradient(to right, $palepink_color, 5%, white);
+    padding-top: 2rem; 
+    padding-bottom: 2rem;
+  }
+
+  .blog_post{
+    margin: 1rem;
+  }
+  //Overlay effect on post images
+
+  .image {
+    display: block;
+    width: 100%;
+    height: auto;
+  }
+  
+  .overlay {
+    position: absolute;
+    top: 0;
+    bottom: 0;
+    left: 0;
+    right: 0;
+    height: auto;
+    width: 100%;
+    opacity: 0;
+    transition: .5s ease;
+    background-color: $darkblue_color;
+  }
+  
+  .container:hover .overlay {
+    opacity: 0.9;
+  }
+  
+  .text {
+    font-size: 1.5rem;
+    font-family: $serif;
+    color: white;
+    padding: 2px;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    -webkit-transform: translate(-50%, -50%);
+    -ms-transform: translate(-50%, -50%);
+    transform: translate(-50%, -50%);
+    text-align: center;
+  }
+
+  .text_link {
+    font-family: $sans-serif;
+    padding: 5px;
+    font-size:1rem;
+    color: $white_color;
+    background-color: $darkblue_color;
+    position: absolute;
+    top: 80%;
+    left: 50%;
+    -webkit-transform: translate(-50%, -50%);
+    -ms-transform: translate(-50%, -50%);
+    transform: translate(-50%, -50%);
+    text-align: center;
+  }
+
+  .text_link:hover {
+    color: $darkblue_color;
+    text-decoration: none;
+    background-color: $palepink_color;
+    
+
+    
+  }
+  
+
+  // Small devices (tablets, 768px and up) 
+
+  .post_content{
+    padding-top: 0;
+    padding-bottom: 1rem;
+  }
+
+  // Large devices (desktops, 992px and up)
+    @media (min-width: 992px) {
+      
+      .post_content{
```

Comment:
Don't need, even should probably, not add the class name again under a media query

Reply:
I am not sure what you mean, if I remove 
.post_content{
        padding-bottom: 10rem;
      }
from desktop view than the text and the image wil de too close to blue text divider in that view.
<img width="856" alt="screenshot 2018-12-12 at 12 57 34" src="https://user-images.githubusercontent.com/44289367/49868395-9bfed800-fe0d-11e8-8056-4bd7d54c59c2.png">